frontend/src/components/Drawer.vue

```diff
@@ -21,6 +21,9 @@
       <section class="p-4">
         <slot name="body"></slot>
       </section>
+      <footer class="absolute bottom-5 w-full">
```

Comment:
I think when the content is full, then the footer will cover it. Please take a testing for the layout.

Reply:
It’s already on my plan: BYT-876, I’ll enhance the drawer later. But before that I would first complete tasks with higher priorities.